README.md

```diff
@@ -15,13 +15,16 @@ A collection of calcite components for building single page applications.
 - [calcite-action-bar](/src/components/calcite-action-bar/readme.md)
 - [calcite-action-pad](/src/components/calcite-action-pad/readme.md)
 - [calcite-block](/src/components/calcite-block/readme.md)
+- [calcite-filter](/src/components/calcite-filter/readme.md)
```

Comment:
I believe we wanted to hold off on advertising the filter as a public component for now. @driskull @jcfranco that right?